README.md

```diff
@@ -53,7 +74,7 @@ player.pause()
 player.quit()
 ```
 
-Playing a file from a URL (courtesy of @jappe999)
+Playing a file from a URL by downloading the network file (courtesy of @jappe999)
```

Comment:
I don't see a clear need for that example either.

Maybe the corner case that someone has a slow connection and wants to download the file completely before starting to play it. But still, I think that shouldn't be a problem that omxplayer-wrapper should "solve" and document.

I'm removing it too.